.laminas-ci.json

```diff
@@ -0,0 +1,6 @@
+{
+    "exclude": [
+        {"name": "PHPUnit on PHP 7.1 with locked dependencies"},

```

Comment:
I'd prefer having support for PHP 7.1 just dropped here.
We can have `^7.3` in `composer.json` to avoid this whole file.

The `composer.lock` then should be created with PHP 7.3.